workers/unity/Packages/io.improbable.worker.sdk.platform/.gitignore

```diff
@@ -0,0 +1,7 @@
+# Ignore hidden folders
+/.*/
+
+# Ignore everything besides .meta in Plugins
+Plugins/**/*.*
+!Plugins/**/*.meta
+!.npmignore
```

Comment:
This needs an npmignore file, otherwise the package wont include the meta files.

Reply:
You can check for correctness by using `npm pack` in the package folder, and see if the resulting tar includes the right files.

Reply:
Yeah I know 😉 I just threw this up as a proof of concept so didn't do the entire process.

I don't think this is strictly merge-able in its current state given that we are compiling a pre-release version of the Platform SDK.